.github/workflows/docs_ci.yml

```diff
@@ -16,3 +16,31 @@ jobs:
         run: ./v check-md -hide-warnings .
       ## NB: -hide-warnings is used here, so that the output is less noisy,
       ## thus real errors are easier to spot.
+
+  report-missing-fn-doc:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build V
+        run: make
+
+      - name: Checkout previous v
+        uses: actions/checkout@v2
+        with:
+          repository: vlang/v
+          path: pv
+
+      - name: Check against parent commit
+        run: |
+          ./v run cmd/tools/missdoc.v ./vlib/ > /tmp/n_v.txt
+          ./v run cmd/tools/missdoc.v ./pv/vlib/ > /tmp/o_v.txt
+          count_new=$(cat /tmp/n_v.txt | wc -l)
+          count_old=$(cat /tmp/o_v.txt | wc -l)
+          if [[ ${count_new} -gt ${count_old} ]]; then
+            echo "The following $((count_new-count_old)) function(s) is introduced with no documentation:"
+            sed -n 's/^.*\/vlib\///p' /tmp/n_v.txt > /tmp/tn_v.txt
+            sed -n 's/^.*\/vlib\///p' /tmp/o_v.txt > /tmp/to_v.txt
+            sort /tmp/tn_v.txt /tmp/to_v.txt | uniq -u
+            exit 1
```

Comment:
why not simply `diff /tmp/tn_v.txt /tmp/to_v.txt` ?

Reply:
I couldn't get diff to work locally - I'm testing with this bash script:
```bash
v run ~/Projects/v/cmd/tools/missdoc.v ~/Projects/v/vlib/ > /tmp/n_v.txt
v run ~/Projects/v/cmd/tools/missdoc.v /tmp/v/vlib/ > /tmp/o_v.txt

count_new=$(cat /tmp/n_v.txt | wc -l)
count_old=$(cat /tmp/o_v.txt | wc -l)
if [[ ${count_new} -gt ${count_old} ]]; then
	echo "The following $((count_new-count_old)) function(s) is introduced with no documentation:"
	sed -n 's/^.*\/vlib\///p' /tmp/n_v.txt > /tmp/tn_v.txt
	sed -n 's/^.*\/vlib\///p' /tmp/o_v.txt > /tmp/to_v.txt
	#sort /tmp/tn_v.txt /tmp/to_v.txt | uniq -u
	diff /tmp/tn_v.txt /tmp/to_v.txt
	exit 1
fi
```
And diff outputs a huge list - which I can't figure out why :(

Reply:
Ah it's because the list isn't sorted - that'll just introduce an extra `sort` step instead

Reply:
I still have problems getting it right even with sort involved

Reply:
I was just curious - if `uniq -u` works better - so be it

Reply:
Ah got it now - hold on

Reply:
Pushed this in b859d9c:

```bash
v run ~/Projects/v/cmd/tools/missdoc.v ~/Projects/v/vlib/ > /tmp/o_v.txt # Local testing
v run ~/Projects/v/cmd/tools/missdoc.v /tmp/v/vlib/ > /tmp/n_v.txt # local testing
#./v run cmd/tools/missdoc.v ./vlib/ > /tmp/n_v.txt
#./v run cmd/tools/missdoc.v ./pv/vlib/ > /tmp/o_v.txt
count_new=$(cat /tmp/n_v.txt | wc -l)
count_old=$(cat /tmp/o_v.txt | wc -l)
if [[ ${count_new} -gt ${count_old} ]]; then
  echo "The following $((count_new-count_old)) function(s) is introduced with no documentation:"
  sed -n 's/^.*\/vlib\///p' /tmp/n_v.txt | sort > /tmp/tn_v.txt
  sed -n 's/^.*\/vlib\///p' /tmp/o_v.txt | sort > /tmp/to_v.txt
  #sort /tmp/tn_v.txt /tmp/to_v.txt | uniq -u
  diff /tmp/tn_v.txt /tmp/to_v.txt
  exit 1
fi
```

Reply:
The output is a little different now (includes a has value???):
```
The following 1 function(s) is introduced with no documentation:
541d540
< os/notify/notify.v:38:0:pub fn undoc() string
```

Reply:
Ah that's just a diff-thing :+1: